src/main/java/com/ait/lienzo/client/widget/panel/event/LienzoPanelScaleChangedEvent.java

```diff
@@ -0,0 +1,34 @@
+package com.ait.lienzo.client.widget.panel.event;
+
+import com.ait.lienzo.client.core.types.Point2D;
+import com.ait.lienzo.client.widget.panel.AbstractLienzoPanelEvent;
+
+public class LienzoPanelScaleChangedEvent
+        extends AbstractLienzoPanelEvent<LienzoPanelScaleChangedEventHandler>
+{
+    public static final Type<LienzoPanelScaleChangedEventHandler> TYPE = new Type<LienzoPanelScaleChangedEventHandler>();
```

Comment:
Explicit type argument `LienzoPanelScaleChangedEventHandler` can be replaced with <>

Reply:
done